ff/src/biginteger/macros.rs

```diff
@@ -168,13 +168,32 @@ macro_rules! bigint_impl {
 
             #[inline]
             fn to_bits(&self) -> Vec<bool> {
-                let mut res = Vec::with_capacity(256);
+                let mut res = Vec::with_capacity($num_limbs * 64);
```

Comment:
Separately, I think we can replace this with `BigIteratorBE::new(self.0).collect::<Vec<_>>()`

Reply:
Oh true, I'll change it to that